src/spyglass/lfp/lfp_merge.py

```diff
@@ -0,0 +1,53 @@
+import datajoint as dj
+from spyglass.lfp.v1 import LFP, ImportedLFP
+
+schema = dj.schema("lfp")
+
+
```

Comment:
I also included a global list called [_valid_data_sources](https://github.com/LorenFrankLab/spyglass/blob/011dcaaca1f1dc20f1cc75c0edd8f2a85ad48ce4/src/spyglass/position/position_merge.py#L21) with the strings of each upstream source that was allowed to be used.